internal/spokes/spokes.go

```diff
@@ -396,9 +409,34 @@ func (r *SpokesReceivePack) readPack(ctx context.Context, commands []command, ca
 		_ = eg.Wait()
 	}
 
+	if waitErr := cmd.Wait(); waitErr != nil {
+		return waitErr
+	}
+
 	return nil
 }
 
+func (r *SpokesReceivePack) isFsckConfigEnabled() bool {
+	receiveFsck := config.GetConfigEntryValue(r.repoPath, "receive.fsckObjects")
+	transferFsck := config.GetConfigEntryValue(r.repoPath, "transfer.fsckObjects")
+
+	if receiveFsck != "true" || transferFsck != "true" {
```

Comment:
Shouldn't this be 
```suggestion
	if receiveFsck == "true" || transferFsck == "true" {
```
?